package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "discord.js": "^11.4.2",
     "luxon": "^1.11.4",
+    "mongoose": "^5.5.2",
```

Comment:
Let's not import more dependencies until they're needed. Flatfiles should suffice for now.